app/models/school.rb

```diff
@@ -10,4 +11,21 @@ def full_name(delim = ', ')
     n += "#{delim}#{borough}" if !borough.nil?
     n
   end
+
+  def activation_links_container(async_boolean = false)
```

Comment:
We shouldn't be generating html through the model. I'd suggest creating a partial view and embedding this information there.